src/instructlab/configuration.py

```diff
@@ -288,6 +288,7 @@ def api_base(self):
 class _mmlu(BaseModel):
     few_shots: int
     batch_size: int
+    device: str
```

Comment:
Is it going to be required here and not backward compatible?

Reply:
Can we move all defaults from `get_default_config` into the model definition?